server/views/formPages/incidentMacros.njk

```diff
@@ -99,7 +99,7 @@
           {% endfor %}
 
           {# followup #}
-          <div class="govuk-checkboxes__conditional clear-both" id="conditional-{{name}}">
+          <div class="govuk-radios__conditional clear-both" data-module="govuk-checkboxes" id="conditional-{{name}}">
```

Comment:
this doesn't look right - why use a `radios` class for `checkboxes`?

Reply:
Changed from checkboxes to radios, even though component is in fact a checkbox, because the component didn't function with the new version of the toolkit. 
This is a non-standard Govuk component